x/authz/client/cli/tx.go

```diff
@@ -160,6 +160,10 @@ Examples:
 				return fmt.Errorf("invalid authorization type, %s", args[1])
 			}
 
+			if err = authorization.ValidateBasic(); err != nil {
+				return err
```

Comment:
I don't think this should be checked in the CLI, but deeper in the stack. Maybe in `func (msg MsgGrantAuthorizationRequest) ValidateBasic()`?

Then `svcMsgClientConn` checks ValidateBasic on msgs.